x-pack/plugins/alerting/server/alert/alert.ts

```diff
@@ -76,6 +77,10 @@ export class Alert<
     return this.meta.uuid!;
   }
 
+  getStart(): string | null {
+    return this.state.start ? (this.state.start as string) : null;
```

Comment:
feels like `${this.state.start}` would be safer than `this.state.start as string`